docs/user-guide/scheduled-jobs.md

```diff
@@ -0,0 +1,177 @@
+---
+assignees:
+- erictune
+- soltysh
+- janetkuo
+
+---
+
+* TOC
+{:toc}
+
+## What is a _Scheduled Job_?
+
+A _Scheduled Job_ manages time based [Jobs](/docs/user-guide/jobs/), namely:
+
+* Once at a specified point in time
+* Repeatedly at a specified point in time
+
+One ScheduledJob object is like one line of a _crontab_ (cron table) file. It runs a job periodically
+on a given schedule, written in [Cron](https://en.wikipedia.org/wiki/Cron) format.
+
+A typical use case is:
+
+* Schedule a job execution at a given point in time.
+* Create a periodic job, e.g. database backup, sending emails.
+
+### Prerequisites
+
+You need a working Kubernetes cluster at version >= 1.4, with batch/v2alpha1 API turned on by passing
+`--runtime-config=batch/v2alpha1` while bringing up the API server (see [Turn on or off an API version
+for your cluster](/docs/admin/cluster-management/#turn-on-or-off-an-api-version-for-your-cluster) for
+more). You cannot use Scheduled Jobs on a hosted Kubernetes provider that has disabled alpha resources.
+
+## Creating a Scheduled Job
+
+Here is an example Scheduled Job. Every minute, it runs a simple job to compute π to 2000 places and prints it out.
```

Comment:
The sample job prints date and `Hello from the Kubernetes cluster` not computes PI.